pkg/util/inplaceupdate/inplace_update_test.go

```diff
@@ -24,13 +24,16 @@ import (
 	"time"
 
 	appspub "github.com/openkruise/kruise/apis/apps/pub"
+	"github.com/openkruise/kruise/pkg/features"
```

Comment:
plz add ut for defaultPatchUpdateSpecToPod

Reply:
done